Sources/SwiftKuery/ConnectionPool.swift

```diff
@@ -23,8 +23,26 @@ import Dispatch
 
 // MARK: ConnectionPool
 
-/// A connection pool implementation.
-/// The pool is FIFO.
+/**
+ This class implements a First in First out connection pool. The pool maintains a cache of `ConnectionPoolConnection` instances, which can be reused for future requests, to enhance performance.
+ ### Usage Example: ###
+ In this example, a closure to generate and release `Connection` instances are defined.
```

Comment:
This needs to use "is" not "are" i.e. "a closure to generate and release `Connection` instances is defined".